cts/scheduler/summary/cancel-behind-moving-remote.summary

```diff
@@ -187,10 +162,10 @@ Revised Cluster Status:
       * haproxy-bundle-podman-1	(ocf:heartbeat:podman):	 Started controller-0
       * haproxy-bundle-podman-2	(ocf:heartbeat:podman):	 Started controller-1
     * Container bundle set: ovn-dbs-bundle [cluster.common.tag/rhosp16-openstack-ovn-northd:pcmklatest]:
-      * ovn-dbs-bundle-0	(ocf:ovn:ovndb-servers):	 Unpromoted controller-2
-      * ovn-dbs-bundle-1	(ocf:ovn:ovndb-servers):	 Promoted controller-0
+      * ovn-dbs-bundle-0	(ocf:ovn:ovndb-servers):	 Unpromoted controller-0
+      * ovn-dbs-bundle-1	(ocf:ovn:ovndb-servers):	 Promoted controller-2
       * ovn-dbs-bundle-2	(ocf:ovn:ovndb-servers):	 Unpromoted controller-1
-    * ip-172.17.1.87	(ocf:heartbeat:IPaddr2):	 Started controller-0
+    * ip-172.17.1.87	(ocf:heartbeat:IPaddr2):	 Stopped
```

Comment:
There's a mandatory colocation constraint for `ip-172.17.1.87` with the promoted role of `ovn-dbs-bundle`. As we know (T672), promotable bundle colocations don't work correctly, so it may not be influencing where the bundle gets promoted (if it's even supposed to).

Additionally, `ip-172.17.1.87` is banned from `controller-2` altogether. So if `ovn-dbs-bundle` decides to promote there, `ip-172.17.1.87` must stop.

The key change in this test output is that the bundle replicas no longer get shuffled.

Reply:
These colocation bugs really needs to be fixed before we accept any fix for the shuffling issue. I remember now that this is why I paused the previous PR attempt. The core fix **probably** works (in that we preferentially don't shuffle instances), but this test represents a regression. An instance that previously would be running is now stopped.